front/src/views/recovery-password/RecoveryPassword.html

```diff
@@ -0,0 +1,162 @@
+<v-card color="#F6F8FB" width="560px" class="mx-auto" elevation="0">
```

Comment:
about color - is it only for your card? if no, might it is better to make it is via common styles?

Reply:
it is possible that the login has the same color, but I am not very sure about it. perhaps I'll put it in common. css